content/install_maintain/installation/installing-cluster.md

```diff
@@ -22,7 +22,7 @@ before you install Cloudify Manager and that you have read the [installation and
 
 ![Cloudify_Cluster]( /images/cluster/cluster-architecture.png )
 
-Cloudify Manager 5.0.5 introduces a new cluster architecture to Cloudify. This cluster is comprised of 3 separate services that construct the entire Cloudify solution:  
+Cloudify Manager 5.1 clusters are composed of three separate services that construct the entire Cloudify solution:  
```

Comment:
According to Moshe's comment at the lecture on Thursday, we should add an introduction to the Cloudify cluster. In my opinion, this introduction should include the advantages of using a Cloudify cluster and an overview of the different services and they way they communicate. @ofercloudify @alexmolev what do you think?